src/components/LayoutHome.vue

```diff
@@ -1,22 +1,22 @@
 <template>
+  <div class="">
+    <div class="fixed h-screen top-0 p-4 min-w-[max-content] w-auto bg-faintGrey text-[16px] font-[500] leading-[24px] rounded-[4px] mb-[8px]  ">
     <div class="">
-      <div class="fixed h-screen top-0 p-8  min-w-[max-content] w-auto bg-faintGrey text-[16px] font-[500] leading-[24px] rounded-[4px] mb-[8px] shadow-lg ">
-      <div class="">
-        <botton class="fixed right-430 cursor-pointer" @click="showComponent = !showComponent" >
-          <img src="../assets/images/svgs/align-right.svg" alt="Image" class="w-[24px] h-[24px] mb-4" />
-        </botton>
-        </div>
-       <div class="flex items-center mt-10">
-         <!-- <h2 class="text-[30px] leading-[48px]">Memo</h2> -->
-       </div>
-        <ul class="w-[max-content] z-[1000] transition-all translate-x-[-360px] md:translate-x-0'" :class="showComponent ? 'translate-x-[-360px]' : 'translate-x-0'">
-        <li v-for="item in keyList" :key="item.id" @click="goAnchor(item)" class="cursor-pointer text-slate-500 text-[18px] font-normal leading-[24px] rounded-[4px] mb-[8px] hover:underline hover:text-black">{{ item.name }}</li>
-       </ul>
+      <botton class="fixed right-430 cursor-pointer" @click="showComponent = !showComponent" >
+        <img src="../assets/images/svgs/align-right.svg" alt="Image" class="w-[24px] h-[24px] mb-4" />
+      </botton>
       </div>
+     <div class="flex items-center mt-10">
+       <!-- <h2 class="text-[30px] leading-[48px]">Memo</h2> -->
      </div>
-      <div class="ml-80 w-3/4" :class="showComponent ? 'translate-x-[-140px]' : 'translate-x-0'">
-       <slot></slot>
-     </div>
+      <ul class=" fixed h-screen shadow-lg w-[max-content] z-[1000] transition-all translate-x-[-360px] md:translate-x-0'" :class="showComponent ? 'translate-x-[-360px]' : 'translate-x-0' ">
```

Comment:
This is not what I asked you to do with the sidebar. The sidebar should not change from the original design and can only be toggled when the device is a tablet or mobile.
Revert the sidebar to the original sidebar, this one is unacceptable.